static/game/chase.js

```diff
@@ -6,6 +6,20 @@ import {
 import { Vector } from "../modules/vector.js";
 import { circle, ellipse, line, polygon } from "./draw.js";
 import { Enemy } from "./enemy.js";
+import { ChanceTable } from "../modules/chancetable.js";
+import { PowerUp } from "./powerup.js";
+import * as PowerUpTypes from "../game/powerups/poweruptypes.js";
+
+const chanceTable = new ChanceTable();
```

Comment:
No type annotation?

Reply:
It knows its a chance table because it's being assigned right there. I would include type annotation if I had to say `const chanceTable` but assign it later, like in a constructor.